integration

```diff
@@ -702,6 +702,7 @@
   "Mat931/digitalstrom-homeassistant",
   "MateoGreil/homeassistant-comwatt",
   "mathieu-mp/homeassistant-intex-spa",
+  "MattDahEpic/ha-midas",
```

Comment:
<https://github.com/MattDahEpic/ha-midas/actions/runs/13611628259/job/38049578317#step:4:4>
![image](https://github.com/user-attachments/assets/0643b730-ec3c-435d-b753-f27826309918)
....

Reply:
The ignore key has been removed and I've updated the link in the PR description